galaxy.yml

```diff
@@ -0,0 +1,51 @@
+### REQUIRED
+
+# The namespace of the collection. This can be a company/brand/organization or product namespace under which all
+# content lives. May only contain alphanumeric characters and underscores. Additionally namespaces cannot start with
+# underscores or numbers and cannot contain consecutive underscores
+namespace: ara
```

Comment:
This might need to be ansible-community ? I'll find out if it's okay to publish from here.